armi/reactor/components/basicShapes.py

```diff
@@ -128,6 +131,10 @@ def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
         sideLength = self.getDimension("op", Tc, cold) / math.sqrt(3)
         return 2.0 * sideLength
 
+    def getBoundingCircleInnerDiameter(self, Tc=None, cold=False):
+        sideLength = self.getDimension("ip", Tc, cold) / math.sqrt(3)
```

Comment:
On second examination, it needs to work like this for your functions to work. so lets remove the word bounding since the circle will pass through he component. or call it 

getInteriorCornerCircleDiameter

Reply:
Addressed. https://github.com/terrapower/armi/pull/670/commits/81318e16eddf5fa360597ee3cf1c148fcc72a560